web/electron/tray-menu.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2021 the Octant contributors. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+import { app, BrowserWindow, Menu, Tray, nativeImage, shell } from 'electron';
+import { errLogPath, iconPath } from './paths';
+
+export class TrayMenu {
+  public readonly tray: Tray;
+
+  constructor(public window: BrowserWindow) {
+    this.tray = new Tray(this.createNativeImage());
+    this.tray.setContextMenu(this.createMenu(window));
+  }
+
+  createNativeImage(): Electron.NativeImage {
+    const image = nativeImage.createFromPath(iconPath);
+    image.setTemplateImage(true);
+    return image;
+  }
+
+  createMenu(win: BrowserWindow): Menu {
```

Comment:
The tray menu does not look good, at least on Mac, see the screenshot.
![Screen Shot 2021-02-01 at 7 59 43 AM](https://user-images.githubusercontent.com/34245594/106477296-3cd50500-6465-11eb-879e-1bd189923ed5.png)

Reply:
😆 glad we have at least one person on a Mac